services/workflows-service/src/data-migration/scripts/sync/sync.ts

```diff
@@ -281,6 +330,30 @@ export const sync = async (objectsToSync: SyncedObject[]) => {
     throw err;
   }
 };
+function mergeEnvironmentConfigs(acc: any, key: any, obj: any) {
+  acc[key].columns = {
+    ...acc[key].columns,
+    ...obj.columns,
+  } as Partial<
+    | WorkflowDefinitionPayload['scalars']
+    | UiDefinitionPayload['scalars']
+    | AlertDefinitionPayload['scalars']
+  >;
+  acc[key].syncedEnvironments = [
+    ...new Set([...acc[key].syncedEnvironments, ...obj.syncedEnvironments]),
+  ];
```

Comment:
We have `uniqueArray` in `@ballerine/common`.

```suggestion
  acc[key].syncedEnvironments = uniqueArray([
   ...acc[key].syncedEnvironments,
    ...obj.syncedEnvironments,
  ]);
```